pages/product-selection-opensuse-page.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+
+export class ProductSelectionOpensusePage {
+    readonly page: Page;
+    readonly productSelectionText: Locator;
+    readonly opensuseTumbleweedLabel: Locator;
+    readonly opensuseLeapLabel: Locator;
+    readonly selectButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.productSelectionText = page.getByText('Product selection');
+        this.opensuseTumbleweedLabel = page.getByLabel('openSUSE Tumbleweed');
+        this.opensuseLeapLabel = page.getByLabel('openSUSE Leap 16.0');
+        this.selectButton = page.getByRole('button', { name: 'Select' });
+    }
+
+    async expectProductSelection() {
```

Comment:
this might not be used now, right? please drop it if it is the case.

Reply:
Removed